src/progressbar/progressbar.ts

```diff
@@ -23,18 +23,18 @@ export class NgbProgressbar {
   /**
    * Maximal value to be displayed in the progressbar.
    */
-  @Input() max = 100;
+  @Input() max: number;
```

Comment:
No. The api-docs component now gets the default value here, and if undefined, looks for it in the matching  property of the associated config service. So the default value should in fact not be defined here:
 - it's useless code, since it's immediately overwritten by the constructor, which copies the default value from the property
 - it's misleading code, since it makes the reader think that the default value is 100, whereas it's in fact copied from the config property in the constructor
 - it's also misleading because the generated documentation reads the default value from here although the actual default value comes from the config property

The last point makes me realize that the api-docs component should do the opposite of what it does now: try to find the default value in the matching config property, and only if not found, read it from here.

I'll amend the PR.